Assets/Expansion1/Replacements/worldtracker_expansion1.lua

```diff
@@ -1,3 +1,6 @@
+-- TEMP
```

Comment:
making a note here as I appreciate this work but want to focus on a single branch if we can, so this note exists only to block the PR until we figure out issues and determine if this should be merged in (I don't want this to be merged before we all figure it out)